### PR TITLE
The DBGet method is asynchronous

### PR DESCRIPTION
**Pull request by luiz-simples**
was used requestList in DBGet returning first element

### CODE REVIEW

File: ami/db.go [hunk 1]
```diff
@@ -27,10 +27,18 @@ func DBPut(client Client, actionID, family, key, val string) (Response, error) {
...
+	data := dbData{
 		Family: family,
 		Key:    key,
-	})
+	}
+
+	responses, err := requestList(client, "DBGet", actionID, "DBGetResponse", "DBGetComplete", data)
+
+	if err != nil {
+		return nil, err
```
Comment (heltonmarx) [inline ami/db.go]: It's better verify if the response is `len(responses)` greater then zero first, for example:
```go
switch {
  case err != nil :
     return nil, err
 case len(responses) == 0:
     return nil, errors.New("there is no db entries")
}
return responses[0], nil
```

Comment (luiz-simples) [reply ami/db.go]: tks

Comment (heltonmarx) [reply ami/db.go]: I correct one grammar mistake, the correct form is: `there are no db entries`, instead `there is...`

Comment (luiz-simples) [reply ami/db.go]: done

**Pull request merged**